pkg/metrics/metrics.go

```diff
@@ -369,6 +370,27 @@ If the ClusterQueue has a weight of zero, this will return 9223372036854775807,
 the maximum possible share value.`,
 		}, []string{"cluster_queue"},
 	)
+
+	// Metrics counting usage of frameworks
+
+	// Counter of jobs that are managed by Kueue
+	WorkloadIntegrationsCounter = prometheus.NewCounterVec(
```

Comment:
is it really a counter? I'm leaning towards a gauge since the metric doesn't seem to be incremented anywhere.

Reply:
func CountExternalFramework(externalGVK string) {
	ExternalFrameworksWorkloadCounter.WithLabelValues(externalGVK).Inc()
}

func CountFramework(framework string) {
	WorkloadIntegrationsCounter.WithLabelValues(framework).Inc()
}